sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/input.py

```diff
@@ -118,10 +118,10 @@ def __init__(
         :paramtype default: Union[str, int, float, bool]
         :keyword min: The minimum value for the input. If a value smaller than the minimum is passed to the job, the job
             execution will fail.
-        :paramtype min: Union[int, float]
+        :paramtype min: Optional[float]
```

Comment:
Shouldn't these be `Optional[Union[int, float]]`?

Reply:
This is @overload __init__ function, the real __init__ has covered `Optional[Union[int, float]]`:

![image](https://github.com/Azure/azure-sdk-for-python/assets/19650434/545fa807-1cea-4b06-b46b-e868865a5d6e)